packages/frontend/app/services/reporting.js

```diff
@@ -160,27 +160,68 @@ export default class ReportingService extends Service {
     ].concat(mappedResults);
   }
 
+  async getInstructorsArrayResultsForCourse(courseId) {
+    const userInfo = '{ id firstName middleName lastName displayName }';
+    const block = `instructorGroups {  users ${userInfo}}instructors ${userInfo}`;
```

Comment:
shouldn't there be a space in there?

```suggestion
    const block = `instructorGroups {  users ${userInfo}} instructors ${userInfo}`;
```

Reply:
Apparently it's not needed, but I agree it should be there. Nice catch!